File: Python/Packing.py
# Get inputs
A,B,N = map(int,input().split())
ar = list(map(int,input().split()))
# init answer
ans = 0
for x in ar:
	# Check if element x can be packed or not
    if x <= A or x <= B:
        if A > B:
            A -= x
        else:
            B -= x
    else:
        ans += 1
# print ans
print(ans)

File: Cpp/Packing.cpp
#include <iostream>

using namespace std;

int main() {
    // Get inputs
    int A,B,N;
    cin >> A >> B >> N;
    int ar[N];
    for(int i=0;i<N;i++)
        cin >> ar[i];
    // Init ans
    int ans = 0;
    for(int i=0;i<N;i++){
        // Check if the element can be packed
        if(ar[i] <= A || ar[i] <= B){
            if(A>B)
                A -= ar[i];
            else
                B -= ar[i];
        }else{
            ans++;
        }
    }
    // Print answer
    cout << ans;
    return 0;
}
